src/Microsoft.Health.Fhir.Shared.Core/Features/Validation/IProvideProfilesForValidation.cs

```diff
@@ -0,0 +1,13 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Hl7.Fhir.Specification.Source;
+
+namespace Microsoft.Health.Fhir.Shared.Core.Features.Validation
+{
+    public interface IProvideProfilesForValidation : IResourceResolver, ISummarySource
```

Comment:
Perhaps `IValidationProfileProvider` :P

Reply:
TomatO, TOmato, PotatO, POtato...

---
In reply to: [574166445](https://github.com/microsoft/fhir-server/pull/1654#discussion_r574166445) [](ancestors = 574166445)